src/responder/kcm/kcmsrv_cmd.c

```diff
@@ -129,23 +129,29 @@ struct kcm_reqbuf {
     struct kcm_iovec v_msg;
 };
 
+static uint32_t kcm_input_get_payload_len(uint8_t *base,
```

Comment:
Since the base and len parameter should only ever be part of a single kcm_iovec structure, wouldn't it be better to use a struct kcm_iovec as the parameter here?

Reply:
Fixed locally, thanks!